src/meetingsession/DefaultMeetingSession.ts

```diff
@@ -32,15 +29,15 @@ export default class DefaultMeetingSession implements MeetingSession, Destroyabl
   private _logger: Logger;
   private audioVideoController: AudioVideoController;
   private contentShareController: ContentShareController;
+  private _eventController: EventController;
   private _deviceController: DeviceController;
   private audioVideoFacade: AudioVideoFacade;
-  private _eventReporter: EventReporter;
 
   constructor(
     configuration: MeetingSessionConfiguration,
     logger: Logger,
     deviceController: DeviceControllerBasedMediaStreamBroker,
-    eventReporter?: EventReporter
+    eventController?: EventController
```

Comment:
wonder if we could just do private readonly here? Would it be the same as _eventController and a getter?

Reply:
We could do private here, but we can't do readonly because we can't set it to undefined upon destruction.